src/sfizz/OpcodeCleanup.re

```diff
@@ -30,7 +30,7 @@ static std::string cleanUpOpcodeName(absl::string_view rawOpcode, OpcodeScope sc
     const char* YYCURSOR;
     const char* YYMARKER;
     const char* yypmatch[2 * YYMAXNMATCH];
-    size_t yynmatch;
+    //size_t yynmatch;
```

Comment:
This one, it gets used or not depending on parser rules, which I don't remember it exactly.
I'll rather have it as `UNUSED()` instead of commented.
Also an observation, `./scripts/generate_re2c.sh` should be called after editing this one, and post the cpp file changes as well.

Reply:
IDK much how this works, if it has some sort of automation, just simply based on code that it was unused anywhere, so I simply commented it.

Reply:
`OpcodeCleanup.cpp` is generated source based on this file.
In this case, it's fine to include `Macros.h` in the RE file, put `UNUSED(yynmath)` in there, generate cpp, and commit the whole.